docs/source/index.rst

```diff
@@ -1,132 +1,89 @@
+This package can be installed in an activated environment with:
 
 .. code-block:: bash
 
-    python3 bootstrap.py --drivers
+    pip install dnp3-python
 
 
 Driver Configuration
---------------------
+====================
+
+There is one argument for the "driver_config" section of the DNP3 driver configuration file:
 
-There are three arguments for the `driver_config` section of the device configuration file:
+Here is a sample DNP3 driver configuration file:
 
-    - **device_address** - IP Address of the device.
-    - **port** - Port the device is listening on.  Defaults to 502 which is the standard port for Modbus devices.
-    - **slave_id** - Slave ID of the device. Defaults to 0.  Use 0 for no slave.
+.. code-block:: json
 
-The remaining values are as follows:
+    {
+      "driver_config": {
+        "master_ip": "0.0.0.0",
+        "outstation_ip": "127.0.0.1",
+        "master_id": 2,
+        "outstation_id": 1,
+        "port": 20000
+      },
+      "registry_config": "config://dnp3.csv",
+      "driver_type": "dnp3",
+      "interval": 5,
+      "timezone": "UTC",
+      "publish_depth_first_all": true,
+      "heart_beat_point": "random_bool"
+    }
 
+A sample DNP3 driver configuration file can be found in the VOLTTRON repository
+in `example-config/dnp3.config
+<https://github.com/eclipse-volttron/volttron-lib-dnp3-driver/blob/main/example-config/dnp3.config>`_.
 
 
-Here is an example device configuration file:
+DNP3 Registry Configuration File
+================================
 
-.. code-block:: json
+The driver's registry configuration file, a `CSV <https://en.wikipedia.org/wiki/Comma-separated_values>`_ file,
+specifies which DNP3 points the driver will read and/or write. Each row configures a single DNP3 point.
 
-    {
-        "driver_config": {"device_address": "10.1.1.2",
-                          "port": 502,
-                          "slave_id": 5},
-        "driver_type": "modbus",
-        "registry_config":"config://registry_configs/hvac.csv",
-        "interval": 60,
-        "timezone": "UTC",
-        "heart_beat_point": "heartbeat"
-    }
+The following columns are required for each row:
 
-A sample MODBUS configuration file can be found in the VOLTTRON repository in
-`examples/configurations/drivers/modbus.config`
+    - **Volttron Point Name** - The name used by the VOLTTRON platform and agents to refer to the point.
+    - **Group** - The point's DNP3 group number.
+    - **Variation** - THe permit negotiated exchange of data formatted, i.e., data type.
+    - **Index** - The point's index number within its DNP3 data type (which is derived from its DNP3 group number).
+    - **Scaling** - A factor by which to multiply point values.
+    - **Units** - Point value units.
+    - **Writable** - TRUE or FALSE, indicating whether the point can be written by the driver (FALSE = read-only).
 
+Consult the **DNP3 data dictionary** for a point's Group and Index values. Point
```

Comment:
Where does one find a "DNP3 data dictionary"? I assume this is a definition of what the outstation exposes, and the user should already have it? Without a little more context here, though, this sounds like another configuration file which we are failing to explain.